### ISSUE TITLE
The tests for the auth service fail

### ISSUE THREAD

**Issue by GabrielInTheWorld** [OpenSlides/openslides-auth-service#71]
The tests fail because of the following error:

```
Building datastore-reader
unable to prepare context: unable to 'git clone' to temporary context directory: error fetching: /usr/lib/git-core/git-remote-https: /tmp/_MEImuxdft/libcrypto.so.1.1: version `OPENSSL_1_1_1' not found (required by /lib/x86_64-linux-gnu/libssh.so.4)
: exit status 128
Service 'datastore-reader' failed to build
make: *** [Makefile:30: run-tests] Error 1
Error: Process completed with exit code 2.
```

Any ideas?

**Comment by FinnStutzenstein** [comment]
You get the error even befor building the image, right? SO you do not see
```
Building datastore-reader
Step 1/21 : FROM python:3.8.5-slim-buster
 ---> 6cf621cb1327
```

**Comment by FinnStutzenstein** [comment]
Did you update docker or docker-compose recently? Can you provide both versions? Maybe you have he same issue as in https://github.com/OpenSlides/openslides-backend/pull/435

**Comment by GabrielInTheWorld** [comment]
> Did you update docker or docker-compose recently? Can you provide both versions?

How can I update docker or docker-compose for Github CI?
How can I provide _both_ versions? (which _both_ versions?)

**Comment by GabrielInTheWorld** [comment]
Maybe you have the same issue as in [OpenSlides/openslides-backend#435](https://github.com/OpenSlides/openslides-backend/pull/435)

I tried, but that does not solve this issue... :/

**Comment by GabrielInTheWorld** [comment]
> You get the error even befor building the image, right? So you do not see
> 
> ```
> Building datastore-reader
> Step 1/21 : FROM python:3.8.5-slim-buster
>  ---> 6cf621cb1327
> ```

That's right.

**Comment by FinnStutzenstein** [comment]
> How can I provide _both_ versions? (which _both_ versions?)

```
docker --version
docker-compose --version
```

Can you try https://github.com/openssl/openssl/issues/5845

**Comment by GabrielInTheWorld** [comment]
> > How can I provide _both_ versions? (which _both_ versions?)
> 
> ```
> docker --version
> docker-compose --version
> ```
> 
> Can you try [openssl/openssl#5845](https://github.com/openssl/openssl/issues/5845)

My versions are:
Docker version 20.10.6, build 370c28948e
docker-compose version 1.29.1, build unknown

**Comment by radiomime** [comment]
Any resolution here @GabrielInTheWorld ? I'm having the same issue:

**Comment by GabrielInTheWorld** [comment]
> Any resolution here @GabrielInTheWorld ? I'm having the same issue:
> 

Unfortunately not yet. I'm still investigating this problem and try to fix it.

**Comment by GabrielInTheWorld** [comment]
> Any resolution here @GabrielInTheWorld ? I'm having the same issue:
> 
> ```
> unable to prepare context: unable to 'git clone' to temporary context directory: error fetching: /usr/lib/git-core/git-remote-https: /tmp/_MEI68t0fT/libcrypto.so.1.1: version `OPENSSL_1_1_1' not found (required by /lib/x86_64-linux-gnu/libssh.so.4) 
> ```
> 
> ```
> $ docker --version
>   Docker version 20.10.7, build f0df350
> 
> $ docker-compose --version
>   docker-compose version 1.29.2, build 5becea4c
> 
> $ openssl version
>   OpenSSL 1.1.1f  31 Mar 2020
> ```

@radiomime I noticed, that the tests won't be executed on Ubuntu-latest (Ubuntu-20.04) on GitHub-Actions. Instead, they can be executed on Ubuntu-18.04. So, to run docker-compose files on GitHub-Actions use Ubuntu-18.04.

**Comment by GabrielInTheWorld** [comment]
Fixed in #84.
